app/controllers/scaptimony_policies_controller.rb

```diff
@@ -27,16 +30,24 @@ def parse
 
   def create
     @policy = ::Scaptimony::Policy.new(params[:policy])
-    if @policy.save
-      process_success
+    if @policy.current_step.blank? && @policy.save
```

Comment:
imho, its more readable to have a specific method instead of blank, e.g. something like
```ruby
@policy.completed_wizard? && policy.safe
```